README.md

```diff
@@ -2,6 +2,10 @@
 
 This project acts as a template and exemplar to using Playwright, Typescript and the Page Object Model design pattern to test a multipage application.
 
+## Run tests
+
+Run `npx playwright test` to run all tests, there's no need to start up a webserver as that's handled by the Playwright Test framework. See [the Playwright command line docs](https://playwright.dev/docs/test-cli) for more info.
+
```

Comment:
maybe state that this is through a browser and given the headless version also. although you have given the link